src/vs/workbench/contrib/mergeEditor/browser/commands/commands.ts

```diff
@@ -36,7 +36,7 @@ export class OpenMergeEditor extends Action2 {
 			validatedArgs.input2,
 			validatedArgs.output,
 		);
-		accessor.get(IEditorService).openEditor(input, { preserveFocus: true, override: EditorResolution.DISABLED });
+		accessor.get(IEditorService).openEditor(input, { preserveFocus: true });
```

Comment:
@lramos15 I actually forgot what the strategy should be here: if we pass a typed merge editor input, do we disable the override because it is already resolved or does the resolver take care?

Reply:
This is a deprecated pathway, we don't want you to pass typed inputs to `openEditor` anymore and are trying to get rid of them all. For now this goes to the resolver but the end goal is that typed inputs only be usable on the group service or for `replaceEditors`

Reply:
But do we loose information on the way? I am asking because:
* the merge editor input has info that the untyped one does not provide (`detail`)
* when the resolver gets a typed input, does it convert it to untyped only then to convert it back to typed?

Because that would mean we loose the `detail` property...

Reply:
Alternatively we would have to enrich the merge editor input with a way to pass in `detail` for `input1` and `input2`, maybe that is the way to go actually...

Reply:
We do lose info, the merge editor's factory would be the one who has to convert from untyped to typed however they see fit

Reply:
I have pushed a change to support `detail` property and drop usages of deprecated `openEditor` calls.